packages/typography/src/Paragraph/README.mdx

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Are these READMEs going to conflict with the ones still in `component-library`?

Reply:
Oops I was meant to delete those, nice catch